### DISCUSSION TITLE
Accrescent app store

### DISCUSSION THREAD

**Discussion by ellie3-36** [ferraridamiano/ConverterNOW#319] [Ideas]
### Feature description

Accrescent is an Android app store focused on security, privacy, and usability. Would love to see ConverterNOW app available in Accrescent.

Some relevant links:

Official website: https://accrescent.app/

Docs: https://accrescent.app/docs/index.html

Contact: [contact@accrescent.app](mailto:contact@accrescent.app)

### Why do you want this feature?

Because of Accrescent features:
https://accrescent.app/features

**Comment by ferraridamiano** [comment]
I was not aware of accrescent until now. Is it popular? Why not using f-droid?

**Comment by ellie3-36** [comment]
Accrescent is available for [GrapheneOS](https://grapheneos.org/) users.

The Benefits for me would be a few, like these:

I could get ConverterNOW from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

Security:  
[Signed repository metadata](https://accrescent.app/features#signed-metadata)  
[App signing key pinning](https://accrescent.app/features#key-pinning)  
[Name verification](https://accrescent.app/features#name-icon-verification)

Privacy:  
[No remote APK signing](https://accrescent.app/features#no-remote-signing)  
[No accounts](https://accrescent.app/features#no-accounts)

Some additional info:
https://discuss.grapheneos.org/d/14452-how-to-explain-why-accrescent-over-f-droid/31

**Comment by ferraridamiano** [comment]
I'm gonna close this discussion in favor of #327. In this way we can keep the issues in just one place
